docs/attribution/0001-analytics.rst

```diff
@@ -326,6 +326,36 @@ You do NOT need to include ``datalayer-productdownload-init.es6.js`` in the page
 in the site bundle.
 
 
+Default Browser
+~~~~~~~~~~~~~~~
+
+Trigger this event when a user sets their default browser to Firefox. It's an important conversion for us!
+
+.. code-block:: javascript
+
+    window.dataLayer.push({
+        event: 'default_browser_set',
+    });
+
+
+User Scoped Custom Dimensions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using GA4 through GTM there isn’t a way to set user scoped custom dimensions without an accompanying event.
+The custom event we use for this is `dimension_set`.
+
+.. code-block:: javascript
+
+    window.dataLayer.push({
+        event: 'dimension_set',
+        firefox_isDefault: true
```

Comment:
Nit: this is an odd mix of both snake_case and camelCase, is that something GA prescribes? I wondered if we could just stick to one (probably snake_case as that's what is used elsewhere mostly)?